src/models/cv.ts

```diff
@@ -1,6 +1,7 @@
 import { V1TerminalVersion } from 'models/v1/terminals'
 export type V2CVType = '2'
 export type V3CVType = '3'
+export type V2V3CVType = V2CVType | V3CVType
```

Comment:
lol i'm realizing these types are a bit redundant—cv" stands for "contracts version", so the extra V isn't needed

these could be `CV2Type`, `CV2V3Type` etc maybe. same thing with the constants names. maybe out of scope for this PR

Reply:
Yeah haha, gr8 point indeed.

I think CV2V3Type is tight. will change

Reply:
> V2CVType | V3CVType

Yeah I think I got all these, but will check again 👍